commands/operator-sdk/cmd/up/local.go

```diff
@@ -77,10 +79,18 @@ const (
 )
```

Comment:
On line 76: Is `defaultConfigPath` used anywhere?

Reply:
Nope, removed